src/Application/Actions/Hooks/StripePaymentsUpdate.php

```diff
@@ -30,7 +31,7 @@
  *
  * @return Response
  */
-class StripeChargeUpdate extends Stripe
+class StripePaymentsUpdate extends Stripe
```

Comment:
It might be helpful to have a class comment here which separately lists the two Stripe objects involved / listened for? If I was new to Stripe I think I might assume `Payment` was a first class object in their domain based on this class name.

In any case, we should add to the list of specific events above.